### ISSUE TITLE
`No such file or directory: 'requirements.txt'` on pip install

### ISSUE THREAD

**Issue by NicolasMICAUX** [kadir014/pygame-video#2]
```
Collecting pygamevideo
  Downloading pygamevideo-1.0.0.tar.gz (2.6 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-1ro6jb45/pygamevideo_e6ada7d091a54e58a1dbb58db3f3fafe/setup.py", line 28, in <module>
          install_requires=open("requirements.txt", "r").readlines()
      FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

You forgot to add the requirements.txt I think

**Comment by ghost** [comment]
I also have this question. Have you already solved the question? I want know how to solve it @Doujiamu

**Comment by kadir014** [comment]
Fixed in 2.1.0

`pip install pygamevideo==2.1.0`

https://pypi.org/project/pygamevideo/
